src/objects/application.ts

```diff
@@ -1,10 +1,19 @@
 import { Application } from 'pixi.js'
+import { stage } from '@/objects/stage'
 
 export let application: Application
 
-export function createApplication(view: OffscreenCanvas): void {
+export function createApplication(): void {
   application = new Application({
-    view,
     background: '#1099bb',
+    resizeTo: stage,
   })
+
+  stage.appendChild(application.view as HTMLCanvasElement)
```

Comment:
I'm guessing this casting is necessary cause pixi doesn't have types for this?

Reply:
It has a type but it's not compatible with `Node`. They supposedly fixed this in 7.0.3 but didn't seem to work still. 

From my experience almost all of the methods that accept a node aren't happy with perfectly valid elements. So casting here seamed fine.

Reply:
yeah fair enough, d3 often has this problem as well